machine/topology_test.go

```diff
@@ -25,6 +25,88 @@ import (
 	"github.com/google/cadvisor/utils/sysfs/fakesysfs"
 )
 
+func TestPhysicalCores(t *testing.T) {
+	testfile := "./testdata/cpuinfo"
+	testcpuinfo, err := ioutil.ReadFile(testfile)
+	if err != nil {
+		t.Fatalf("unable to read input test file %s", testfile)
+	}
+
+	numPhysicalCores := GetPhysicalCores(testcpuinfo)
+	if numPhysicalCores != 6 {
+		t.Errorf("Expected 6 physical cores, found %d", numPhysicalCores)
+	}
+}
+
+func TestPhysicalCoresReadingFromCpuBus(t *testing.T) {
+	cpuBusPath = "./testdata/"
```

Comment:
This variable seems to be unused.

Reply:
`cpuBusPath` is a global variable in machine package, here its overwritten to mock sysfs cpu path which is used in `GetPhysicalCores` function